src/client.ts

```diff
@@ -83,8 +88,18 @@ export default class AlertNotificationClient {
             throw new Error('Configuration must not be an empty object, undefined or null');
         }
 
-        if (!configuration.authentication && !configuration.region) {
-            throw new Error('Authentication and region objects are required');
+        if (!configuration.region) {
+            throw new Error('Region object are required');
+        }
+
+        if (!configuration.authentication && !configuration.destinationConfiguration) {
+            throw new Error('Authentication or Destination configuration object is required');
```

Comment:
`throw new Error('Either Authentication or Destination configuration object must be provided');`

Reply:
The proposed message by Yusmen is not fixed